py3status/modules/bluetooth.py

```diff
@@ -60,14 +61,23 @@ class Py3status:
     """
     # available configuration parameters
     cache_timeout = 10
-    device_separator = '|'
     format = DEFAULT_FORMAT
     format_device = '{name}'
+    format_separator = '\|'
 
     def post_config_hook(self):
-        # Do deprecation stuff here.  Because of the complexity we can't use
-        # the usual deprecation methods
-        # THIS IS A SPECIAL CASE DO NOT USE AS EXAMPLE CODE.
+        self.format_separator = self.py3.safe_format(self.format_separator)
```

Comment:
> Dynamic in the future? 

maybe `[\?color=random |]`  where `color_random` is what you can imagine so always changing, hence future

> What about `if old_seperator: else, load self.format_separator`

I don't understand :) all I say is overwriting class variables like this is generally a bad thing and will cause you pain when you least expect it